### PR TITLE
Add compare

### PR DESCRIPTION
**Pull request by heiglandreas**
Introduces a new `compare`-command and does a bit of refactoring.

Thanks to @raphaelstolt (PR #9) for the idea to rename to compare.

### CODE REVIEW

File: app/junitdiff [hunk 1]
```diff
@@ -38,4 +38,5 @@ $app->setVersion('0.2.1');
 $app->setName('JUnitDiff (by Andreas Heigl and contributors)');
 
 $app->add(new \Org_Heigl\JUnitDiff\Command\DiffCommand());
```
Comment (raphaelstolt) [inline app/junitdiff]: What's the difference between these two commands? Shouldn't there only be one command?

File: src/Command/CompareCommand.php [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\Output;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CompareCommand extends Command
+{
+    protected function configure()
+    {
+        $this->setName('compare')
+             ->setDescription('Compare two JUnit log files')
```
Comment (raphaelstolt) [inline src/Command/CompareCommand.php]: Should be in an active _voice_ i.e. `Compares two Junit log files`. See https://github.com/heiglandreas/JUnitDiff/pull/8.

File: app/junitdiff [hunk 3]
```diff
@@ -38,4 +38,5 @@ $app->setVersion('0.2.1');
...
+$app->add(new \Org_Heigl\JUnitDiff\Command\CompareCommand());
```
Comment (raphaelstolt) [inline app/junitdiff:41]: What's the difference between these two commands? Shouldn't there only be one command?

File: composer.json [hunk 4]
```diff
@@ -30,6 +30,6 @@
         "mockery/mockery": "^0.9.5"
     },
     "scripts": {
-        "test": "phpunit"
+        "test": "phpunit --coverage-text"
```
Comment (raphaelstolt) [inline composer.json:33]: To enable code coverage reports you might have to re-enable XDebug by removing [this line](https://github.com/heiglandreas/JUnitDiff/blob/master/.travis.yml#L14) in the Travis CI configuration.

File: app/junitdiff (hunk 3 above)
Comment (heiglandreas) [inline app/junitdiff:41]: There's currently only a difference in the implementation. Even though it's declared as alpha I don't want to break the current API. That's why I introduced _both_ endpoints

**Pull request merged**